packages/manager/modules/logs-to-customer/src/data/api/logService.ts

```diff
@@ -0,0 +1,8 @@
+import apiClient from '@ovh-ux/manager-core-api';
+import { Service } from '../types/dbaas/logs';
+
+/**
+ * GET log service infos
+ */
+export const getLogServicev6 = async (serviceName: string) =>
```

Comment:
As there's no need to handle multiple API version, you can remove the suffix
```suggestion
export const getLogService = async (serviceName: string) =>
```